src/whining.js

```diff
@@ -0,0 +1,68 @@
+// Description:
+//   Whining
+//
+//   A hubot script that creates an agenda for Wednesday Whining
+//
+// Configuration:
+//   HUBOT_IDIO_WHINING_BRANCH: optional
+//   HUBOT_IDIO_WHINING_ORG: optional
+//   HUBOT_IDIO_WHINING_REPO: optional
+//   HUBOT_IDIO_WHINING_TOKEN: required
+//
+// Commands:
+//   hubot it's whine time - Push a whining agenda to the docs repo
+//
+// Author:
+//   Mal Graty
+
+'use strict'
+
+const Octokat = require('octokat')
+const octo = new Octokat({token: process.env.HUBOT_IDIO_WHINING_TOKEN})
+
+const branch = process.env.HUBOT_IDIO_WHINING_BRANCH || 'master'
```

Comment:
> I thought we addressed this in the whining session, but it could also be me remembering my own thoughts rather than vocalised discussion

I could probably say the same. 😄 

Agree though, I guess we can always iterate upon this if/when it proves to be a pain and we haven't entirely painted ourselves into a corner with this solution.